app/src/main/java/com/yagni/model/ReadOneOrder.java

```diff
@@ -55,5 +52,4 @@ public void readOneOrder(DbConnection linkDb){
             System.out.println("Oops! The product ID you entered does not exist!");
```

Comment:
The PR I made before this changed product ID to order number. It was merged and set your branch behind the current upstream

Reply:
The declared package "com.yagni.model" does not match the expected package "App.src.main.java.com.yagni.model"Java(536871240)

I pulled everything into VS code to make sure everything was working correctly. I noticed file name changes/package name change etc. There was a bunch of errors w/ the Dbconnection and the overall package.

I couldn't run this new branch locally. Idk if you could screen share and show us that everything is working properly? Think it's important just because of the last issue w/ gradle and file structure is being changed inside this PR. 

Could you move as a draft until we have time to see it working live?
Also there merge conflict w/ ReadOneOrder would need to be addressed.

Reply:
> The declared package "com.yagni.model" does not match the expected package "App.src.main.java.com.yagni.model"Java(536871240)
> 
> I pulled everything into VS code to make sure everything was working correctly. I noticed file name changes/package name change etc. There was a bunch of errors w/ the Dbconnection and the overall package.
> 
> I couldn't run this new branch locally. Idk if you could screen share and show us that everything is working properly? Think it's important just because of the last issue w/ gradle and file structure is being changed inside this PR.
> 
> Could you move as a draft until we have time to see it working live? Also there merge conflict w/ ReadOneOrder would need to be addressed.

Everything works fine after cleaning the Java language server workspace, these issues are all resolved except the merge conflict

Reply:
Sweet, resolving this convo.

For future reference, if having trouble with the package error he mentioned, inside VS code press the F1 key, and type Clean Java language server workspace. This should fix the issue.